addon/templates/components/lazy-render-wrapper.hbs

```diff
@@ -3,8 +3,10 @@
 		passedPropertiesObject=passedPropertiesObject
 		class=class
 		id=id
+		classNames=classNames
+
 		_shouldShowOnRender=_shouldShowOnRender
-    classNames=classNames
+		_isInProcessOfShowing=_isInProcessOfShowing
```

Comment:
If we're passing private properties in to a child component, should they be private? I'd argue that `isInProcessOfShowing` should be a public property *or* a private property on the `{{component}}`.